### PR TITLE
release 3.0.0

### PR DESCRIPTION
**Pull request by jakobbraun**

### CODE REVIEW

File: doc/changes/changes_3.0.0.md [hunk 1]
```diff
@@ -1,4 +1,6 @@
-# Exasol Test Containers 3.0.0, released 2020-07-29
+# Exasol Test Containers 3.0.0, released 2020-09-14
+
+Code name: Container reuse + automatic port detection
```
Comment (morazow) [inline doc/changes/changes_3.0.0.md:3]:
```suggestion
Code name: Container reuse and automatic port detection
```

File: doc/changes/changes_3.0.0.md [hunk 2]
```diff
@@ -8,9 +10,11 @@ Keep in mind that you still need to expose the right port numbers at constructio
 
 We had to make one not-backward compatible interface change. `getDatabaseNames()` on the `ClusterConfiguration` now returns a list instead of a set. This was necessary, because we need to know which the first listed database is, since this is always the one used for the initial start-up check.
 
+
```
Comment (morazow) [inline doc/changes/changes_3.0.0.md:13]:
```suggestion
```

File: pom.xml [hunk 3]
```diff
@@ -4,7 +4,7 @@
     <modelVersion>4.0.0</modelVersion>
     <groupId>com.exasol</groupId>
     <artifactId>exasol-testcontainers</artifactId>
-    <version>3.1.0</version>
+    <version>3.0.0</version>
```
Comment (morazow) [inline pom.xml:7]: So, versions `3.0.0` and `3.1.0` were not released before?

**Review by morazow — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 3 above)
Comment (jakobbraun) [reply pom.xml:7]: Yes. I did just increase the version number and did not recognize that 3.0.0 was'n released

**Review by morazow — APPROVED** [review]

**Pull request merged**